Gems/Atom/Feature/Common/Code/Source/CoreLights/DirectionalLightFeatureProcessor.cpp

```diff
@@ -1628,6 +1649,19 @@ namespace AZ
             }
         }
 
+        void DirectionalLightFeatureProcessor::SetFullscreenPassSettings()
+        {
+            if (m_fullscreenShadowPass)
+            {
+                const uint32_t shadowFilterMethod = m_shadowData.at(nullptr).GetData(m_shadowingLightHandle.GetIndex()).m_shadowFilterMethod;
+                const uint32_t cascadeCount = m_shadowData.at(nullptr).GetData(m_shadowingLightHandle.GetIndex()).m_cascadeCount;
+                m_fullscreenShadowPass->SetLightIndex(m_shadowingLightHandle.GetIndex());
+                m_fullscreenShadowPass->SetBlendBetweenCascadesEnable(cascadeCount > 1 && m_shadowProperties.GetData(m_shadowingLightHandle.GetIndex()).m_blendBetwenCascades);
+                m_fullscreenShadowPass->SetFilterMethod(static_cast<ShadowFilterMethod>(shadowFilterMethod));
+                m_fullscreenShadowPass->SetReceiverShadowPlaneBiasEnable(m_shadowProperties.GetData(m_shadowingLightHandle.GetIndex()).m_isReceiverPlaneBiasEnabled);
```

Comment:
nit: having a bunch of separate setters exposed which are all just called one after the other doesn't seem like a great pattern. If all of this stuff is expected to be set as part of initialization, I think it'd be cleaner to have a single `Initialize()` function that just takes a descriptor. This would make more sense if those properties were individually set from different places in the code.

Reply:
Ideally the Pass should get the data from the Feature Processor, not the other way around. But it's fine for now, some other passes/FPs already do this kind of caching thing, should do a general pass over them as some point to change it to passes querying FPs